README.md

```diff
@@ -132,7 +144,7 @@ password=changeme
 --row-tag <tag>
 ```
 * Here `row-tag` is a specific option for `raw-format` of type `XML`. For more options for different types please see our WIKI.
-* In case Menas is configured for in-memory authentication (e.g. in dev environments), replace `--menas-auth-keytab` with `--menas-credentials-file`
+* In case REST API is configured for in-memory authentication (e.g. in dev environments), replace `--menas-auth-keytab` with `--menas-credentials-file`
```

Comment:
I wonder if `--menas-auth-keytab` or  `--menas-credentials-file` shouldn't also relate to the _rest-api_ rather than _menas_.